test/screenshotter/ss_data.js

```diff
@@ -0,0 +1,14 @@
+var fs = require("fs");
```

Comment:
`fs` is part of the node standard library and contains a bunch file handling functions.